stories/index.js

```diff
@@ -30,3 +31,11 @@ storiesOf('Upvote', module)
   .add('active', () => (
     <Upvote onClick={action('clicked')} active={true} />
   ))
+
+storiesOf('Points', module)
```

Comment:
Ah, I see what you mean by looking at your PR...so you think I should have added another `.add()` to the `storiesOf('Upvote')` that will show the points count, the way you did with the comments count?